### PR TITLE
Changes required by docc

### PR DESCRIPTION
**Pull request by adam-fowler**

### CODE REVIEW

File: Sources/HummingbirdAuth/Authenticator/LoginCache.swift [hunk 1]
```diff
@@ -28,34 +28,34 @@ public struct LoginCache: Sendable {
     }
 
     /// Logout authenticatable object. Removes object from cache
-    /// - Parameter auth: authentication type
+    /// - Parameter authenticatedType: authentication type
     @inlinable
-    public mutating func logout<Auth: Authenticatable>(_: Auth.Type) {
-        self.cache[ObjectIdentifier(Auth.self)] = nil
+    public mutating func logout<Auth: Authenticatable>(_ authenticatedType: Auth.Type) {
+        self.cache[ObjectIdentifier(authenticatedType)] = nil
```
Comment (Joannis) [inline Sources/HummingbirdAuth/Authenticator/LoginCache.swift:34]: Why did you change this? IIRC passing `Auth.self` is more efficient than passing the metatype value.

File: Sources/HummingbirdAuth/Authenticator/LoginCache.swift [hunk 2]
```diff
@@ -28,34 +28,34 @@ public struct LoginCache: Sendable {
...
     }
 
     /// Return authenticated type
-    /// - Parameter auth: Type required
+    /// - Parameter authenticatedType: Type required
     @inlinable
-    public func get<Auth: Authenticatable>(_: Auth.Type) -> Auth? {
-        return self.cache[ObjectIdentifier(Auth.self)] as? Auth
+    public func get<Auth: Authenticatable>(_ authenticatedType: Auth.Type) -> Auth? {
+        return self.cache[ObjectIdentifier(authenticatedType)] as? Auth
```
Comment (Joannis) [inline Sources/HummingbirdAuth/Authenticator/LoginCache.swift:41]: Same question here about metatypes

File: Sources/HummingbirdAuth/Authenticator/LoginCache.swift [hunk 3]
```diff
@@ -28,34 +28,34 @@ public struct LoginCache: Sendable {
...
     }
 
     /// Require authenticated type
-    /// - Parameter auth: Type required
+    /// - Parameter authenticatedType: Type required
     @inlinable
-    public func require<Auth: Authenticatable>(_: Auth.Type) throws -> Auth {
-        guard let auth = get(Auth.self) else {
+    public func require<Auth: Authenticatable>(_ authenticatedType: Auth.Type) throws -> Auth {
+        guard let auth = get(authenticatedType) else {
```
Comment (Joannis) [inline Sources/HummingbirdAuth/Authenticator/LoginCache.swift:48]: And here

File: Sources/HummingbirdAuth/Authenticator/LoginCache.swift [hunk 4]
```diff
@@ -28,34 +28,34 @@ public struct LoginCache: Sendable {
...
     }
 
     /// Return if cache is authenticated with type
-    /// - Parameter auth: Authentication type
+    /// - Parameter authenticatedType: Authentication type
     @inlinable
-    public func has<Auth: Authenticatable>(_: Auth.Type) -> Bool {
-        return self.cache[ObjectIdentifier(Auth.self)] != nil
+    public func has<Auth: Authenticatable>(_ authenticatedType: Auth.Type) -> Bool {
+        return self.cache[ObjectIdentifier(authenticatedType)] != nil
```
Comment (Joannis) [inline Sources/HummingbirdAuth/Authenticator/LoginCache.swift:58]: And here

**Review by Joannis — CHANGES_REQUESTED** [review]

File: Sources/HummingbirdAuth/Authenticator/LoginCache.swift (hunk 1 above)
Comment (adam-fowler) [reply Sources/HummingbirdAuth/Authenticator/LoginCache.swift:34]: If it is quicker I'll revert. I did this to avoid an annoying swift format/documentation issue where swift format removes the parameter name and docc complains about the parameter name

**Review by Joannis — APPROVED** [review]
